pom.xml

```diff
@@ -56,21 +56,13 @@
             <id>acceleo</id>
             <url>https://repo.eclipse.org/content/repositories/acceleo-releases/</url>
         </repository>
-        <repository>
```

Comment:
why is this repo removed? I think both release/snapshots are enabled by default and this is just an optimisation to reduce HTTP requests.

Reply:
I was cleaning up. It woudl be nice to know where the artefacts are comming from (or not) I thought.